src/mrack/providers/aws.py

```diff
@@ -241,6 +252,7 @@ async def create_server(self, req):
                     },
                 },
             ],
+            "TagSpecifications": {"Tags": taglist},
```

Comment:
ResourceType would be 
```suggestion
            "TagSpecifications": {"ResourceType": "instance",
                                  "Tags": taglist},
```

Reply:
I think it's builds actually failing because it needs that field